README.md

```diff
@@ -15,7 +15,7 @@ A simple canvas component for React Native that allows drawing and signature cap
 
 ## Demo
 
-<img src="https://private-user-images.githubusercontent.com/5255330/437026438-cc1973ff-88d0-46ec-ad8f-1faae985ac4d.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU0OTkzNzgsIm5iZiI6MTc0NTQ5OTA3OCwicGF0aCI6Ii81MjU1MzMwLzQzNzAyNjQzOC1jYzE5NzNmZi04OGQwLTQ2ZWMtYWQ4Zi0xZmFhZTk4NWFjNGQuZ2lmP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDQyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA0MjRUMTI1MTE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NWQ3YzQzYzU2YmI4MzdjMDFiYmUyMjdjMzJjZDMzYWNhYjNlODg4MTAwMzFjYjRjZDQzZjI4ZTk3ZGRjZmNmMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.pJLApYjAmx7GChhQNilSskiVyz1WSrIJmZWjfsBnVZ8" alt="SimpleCanvas Demo" width="50%">
+<img src="https://github.com/guidion-digital/react-native-simple-canvas/blob/acc/437026438-cc1973ff-88d0-46ec-ad8f-1faae985ac4d.gif" alt="SimpleCanvas Demo" width="50%">
```

Comment:
You put the file in the root folder. Can you put it in its own folder? And also give the file a proper name.